### PR TITLE
Refactor/appointment notify

### PR DESCRIPTION
**Pull request by olastor**

### CODE REVIEW

File: src/scripts/appointment-notify.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  const settings = await Settings.findOne();
+
+  if (!settings || !settings.appointmentsTicker) {
+    // ... no admin has subscribed
+    mongoose.connection.close();
+    return;
+  }
+
+  // Search next appointment for today
+  const now = moment.tz('America/Toronto');
```
Comment (binarious) [inline src/scripts/appointment-notify.js]: Another PR made this configurable, didn't it?

**Review by binarious — CHANGES_REQUESTED** [review]

**Comment by binarious** [comment]
Please resolve the conflicts.

**Comment by olastor** [comment]
Conflicts should be resolved

**Comment by olastor** [comment]
Need to rewrite this a bit. Please don't merge yet

**Review by binarious — APPROVED** [review]

**Pull request merged**